src/styles/components/status-bar/styles.less

```diff
@@ -36,6 +51,10 @@
     text-align: left;
     width: @base-spacing-unit * 2;
 
+    &.staged {
+      .staged;
```

Comment:
@weblancaster I'm not sure what happens here - can you elaborate on what's happening for such a declaration?

Reply:
It's basically a `Object.assign({...}, {...})` therefore extending the class `.staged` under `&.staged` because now I have the staged at the wrapper level and the child level (before was only child) so I can re-use the properties/values.